### PR TITLE
feat: integrate create job

### PR DESCRIPTION
**Pull request by maulanasdqn**
**Status** : Done

- [x] Integrate Create Job Action
- [x] Conditional Analyze
- [x] Integrate Loading CV Analyze

image
image
image

### CODE REVIEW

**Comment by maulanasdqn** [comment]
help review this PR @dwicaksono @RikoOrlando

File: lib/actions/job/create-job-server.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
+'use server';
```
Comment (dwicaksono) [inline lib/actions/job/create-job-server.ts:1]: Kenapa di pindah, Buat file baru ? 
Di file Sebelumnya kalau cuman di tambahin "use server" Works ga ?

Comment (maulanasdqn) [reply lib/actions/job/create-job-server.ts:1]: enggak bisa mas, error makanya saya pindahin

Comment (maulanasdqn) [reply lib/actions/job/create-job-server.ts:1]: https://stackoverflow.com/questions/76957592/error-only-async-functions-are-allowed-to-be-exported-in-a-use-server-file

lebih jelasnya disitu mas

**Pull request merged**